src/docs/product/relay/options.mdx

```diff
@@ -16,75 +16,13 @@ All configuration keys are `snake_case`.
-project identifiers can be safely omitted in these cases, and you can continue
-to use the previous DSNs in your services.
+| key                        | type    | value                                                                                         | description                                                                                                                                                                                                                                                                                                                                                                                                                                     |
+| -------------------------- | ------- | --------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| relay.mode                 | String  | default value: `managed` possible values: `managed, static, proxy, capture` optional: `false` | Controls how Relay obtains the project configuration for events. For detailed explanation of these modes, see [Relay Modes](/product/relay/modes/).                                                                                                                                                                                                                                                                                             |
+| relay.upstream             | String  | default value: `https://sentry.io` optional: `false`                                          | Fully qualified URL of the upstream Relay or Sentry instance. <Alert level="warning" title="Important"> Relay does not check for cycles. Do not set this option to an endpoint that will cause events to be cycled back here. </Alert>                                                                                                                                                                                                          |
+| relay.host                 | String  | default value: `0.0.0.0  in Docker, otherwise  127.0.0.1` optional: `false`                   | The host to which Relay should bind (network interface). Example: `0.0.0.0`                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
```suggestion
| relay.host                 | String  | default value: `0.0.0.0  in Docker, otherwise  127.0.0.1` optional: `false`                   | The host to which Relay should bind (network interface). Example: `0.0.0.0`                                                                                                                                                                                                                                                                                                                                                                     |
```
Do we need to add an example since the default value seems to be the same thing?

Reply:
Good point, removed it. I simply copied/pasted the previous info there